src/components/appBootstrap/BrowserPageBootstrap.tsx

```diff
@@ -0,0 +1,130 @@
+/** @jsx jsx */
+import { Amplitude, AmplitudeProvider } from '@amplitude/react-amplitude';
+import { jsx } from '@emotion/core';
+import * as Sentry from '@sentry/node';
+import { createLogger } from '@unly/utils-simple-logger';
+import React from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { userSessionContext } from '../../stores/userSessionContext';
```

Comment:
Just a small thing, when importing files you could you import it like `import stores/userSessionContext`. (https://nextjs.org/blog/next-9-4#absolute-imports-and-aliases)

Reply:
Thanks for the heads-up. I had missed v9.4 release!

I'm not sure about this one, it seems great and all, but it will likely be confusing for newcomers, because it's not widely supported yet and people will likely believe it's loading a package and not a local file.

Reply:
Well this is a common setup for devs that work with webpack and/or babel, maybe not that common for new devs though.

Reply:
@Vadorequest, @samuelcastro
For me is much better to use typescript aliases
```
{
    "baseUrl": ".",
    "paths": {
      "@public/*": [
        "./public/*"
      ],
      "@ts/*": [
        "./src/ts/*"
      ],
      "@components/*": [
        "./src/components/*"
      ],
 ....
}
```
especially for users that uses Webstorm IDE like me with this setup auto-imports will be already aliased + autocompletion works fine. 
![image](https://user-images.githubusercontent.com/18473137/81847068-bfc2d200-955b-11ea-9f33-0d651c8b115a.png)

I don't prefer absolute paths like in CRA and now in next.js 9.4+, because it’s hard for me to immediately understand what it is - my file or the module file. With aliases that starts with @ that seems better for understand.
I prefer specific named aliases

`import someComponent form '@components/someComponent';`

 but for someone maybe helpful this: 
```
"paths": {
      "@/*": [
        "./src/*"
      ],

```

then use in imports @ like src root
`import someComponent form '@/components/someComponent';`

Reply:
Both ways relies on specific configuration and looks like dark magic for beginners. I'm not fan of either, relative paths are easier to understand for everybody, people can always change that afterwards, if they wish so.